cmd/root.go

```diff
@@ -16,51 +16,51 @@ import (
 	"github.com/dhth/omm/internal/ui"
 	"github.com/dhth/omm/internal/utils"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
 )
 
 const (
-	author                = "@dhth"
-	repoIssuesUrl         = "https://github.com/dhth/omm/issues"
-	defaultDataDir        = ".local/share"
-	defaultDataDirWindows = "AppData/Local"
-	dbFileName            = "omm/omm.db"
-	printTasksDefault     = 20
-	taskListTitleMaxLen   = 8
+	defaultConfigFilename   = "omm"
+	envPrefix               = "OMM"
+	author                  = "@dhth"
+	repoIssuesUrl           = "https://github.com/dhth/omm/issues"
+	defaultConfigDir        = ".config"
+	defaultDataDir          = ".local/share"
+	defaultConfigDirWindows = "AppData/Roaming"
+	defaultDataDirWindows   = "AppData/Local"
```

Comment:
Please consider using

[os.UserConfigDir](https://pkg.go.dev/os#UserConfigDir)

[UserCacheDir](https://pkg.go.dev/os#UserCacheDir)

Reply:
Good tip!
I'll change this.